app/src/main/java/de/danoeh/antennapod/fragment/ItemDescriptionFragment.java

```diff
@@ -113,6 +115,7 @@ private void load() {
     public void onPause() {
         super.onPause();
         savePreference();
+        webvDescription.scrollTo(0,0);
```

Comment:
Is this really needed? It will scroll around when the user just turns off the screen for a second.

Reply:
yea it is, without it theres this really strange behaviour that sometimes the description is somewhere in the middle (like this open description & scroll down, close bottomsheet, open bottomsheet, go to description). if the screen is turned off theres no scrollling because backToCover is called. I think this is the expected behaiviour :)